packages/apps/kadena-docs/src/utils/staticGeneration/getData.js

```diff
@@ -0,0 +1,11 @@
+import { menuData } from './../../data/menu.js';
+
+export const getData = () => {
+  const regex = /__tests/;
+  if (process.env.NEXT_PUBLIC_APP_DEV === 'test') {
+    return menuData.filter((item) => regex.test(item.root))[0].children;
```

Comment:
the filter could end up with 0 items, then the `children` will be pointing to an `undefined`

Reply:
it will never be :). but it will be better code. have changed